tests/unit/service/test_files_service.py

```diff
@@ -127,6 +128,24 @@ async def test_upload_paths_to_folder(
         assert Path("tests/data/upload_folder/example.txt") in mocked_upload_file_paths.call_args[1]["file_paths"]
         assert Path("tests/data/upload_folder/example.pdf") in mocked_upload_file_paths.call_args[1]["file_paths"]
 
+    async def test_upload_paths_to_folder_skips_incompatible_file_and_logs_file_name(
+        self,
+        file_service: FilesService,
+        monkeypatch: MonkeyPatch,
+    ) -> None:
+        with capture_logs() as cap_logs:
```

Comment:
With this `capture_logs` method you can test that a certain log line was printed :)